src/Components/Description/Description.js

```diff
@@ -0,0 +1,35 @@
+import { MdOutlineNotStarted } from 'react-icons/md'
+import Card from '../Card'
+import BarUI from '../BarUI'
+
+const Description = () => {
+  return (
+    <section className="description">
+      <div className="description__aside">
+        <BarUI />
+        <h1>The Zabdos Experience</h1>
+        <p className="description__aside--paragraph">
```

Comment:
This is not a proper use of BEM, you cannot use a modifier on its own. It always needs to be connected with the base, so in this case `description__aside description__aside--paragraph` or using "our BEM" `description__aside -paragraph`. But in this case it should simply be `description__paragraph`